src/components/profile/Profile.js

```diff
@@ -27,7 +27,7 @@ const Profile = props => {
       <Section style={styles.section}>
         <Section.Row style={styles.centered}>
           <PrivateIcon onPress={() => log.debug('PrivateIcon')} />
-          <UserAvatar profile={profile} />
+          <UserAvatar profile={profile} source={profile.avatar} />
```

Comment:
I think UserAvatar should only accept profile property
source for Avatar should be using avatar property from profile from inside UserAvatar

Reply:
done